lib/domain/storage/redis.js

```diff
@@ -208,6 +208,12 @@ class StorageRedis extends Storage {
       data.env = JSON.stringify(code.env);
     }
 
+    if (code.exposed === true) {
+      data.exposed = 'true';
+    } else if (code.exposed === false) {
+      data.exposed = 'false';
+    }
```

Comment:
All of this could be:

```javascript
data.exposed = code.exposed.toString();
```

https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Boolean/toString